pyro/infer/inspect.py

```diff
@@ -556,7 +556,7 @@ def render_model(
     model: Callable,
     model_args: Optional[Union[tuple, List[tuple]]] = None,
     model_kwargs: Optional[Union[dict, List[dict]]] = None,
-    filename: Optional[str] = None,
+    filepath: Optional[str] = None,
```

Comment:
Could you keep the signature `filename`?

Reply:
Ok, is it ok now? :)